### PR TITLE
Gessawe in es

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: ES/ESap026.xml [hunk 1]
```diff
@@ -266,11 +365,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                     ʾabuna Gabra Māryām and Walatta Mikāʾel (s. <locus target="#5rc">5rc</locus>, 7rb, 13vb, 15va), 
+                     Gabra Krǝstos and Walatta
+                     Tǝnśāʾe (s. <locus target="#6ra">6ra</locus>, 8va), Takla Giyorgis and Walatta
+                     Māryām (s. <locus target="#6rc">6rc</locus>, 13va), Gabra Māryām and Walatta
+                     Krǝstos (s. <locus target="#6vc">6vc</locus>, 8ra), Gabra Ḫǝywät and ʿĀmata
+                     Śǝllāse (s. <locus target="#7ra">7ra</locus>, 8vc), Gabra Śǝllāse and Walatta
+                     Māryām (s. <locus target="#7rc">7rc</locus>, 8rb), Gabra Walda Garimā and
+                     Walatta ʾƎgziʾ (s. <locus target="#7va">7va</locus>, 8vb), 
+                     ʾabuna Takla ʾAb and Walatta Śǝllāse (s. <locus target="#14va">14va</locus>).</provenance>
                </history>
```
Comment (DariaElagina) [inline ES/ESap026.xml:383]: I am not sure that this information belongs to the provenance.

File: ES/ESap026.xml [hunk 2]
```diff
@@ -266,11 +365,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <history>
                   <origin>
                      <origPlace>
-                        <placeName ref="INS0030AP">Ë&#x81;AddÃ¤qaá¸¥arsi á¹&#x96;Ã¤raqliá¹­os /
-                           AP</placeName>
+                        <placeName ref="INS0030AP"></placeName>
                      </origPlace>
                      <origDate notBefore="1950" notAfter="2000">Second half of 20th cent.</origDate>
```
Comment (DariaElagina) [inline ES/ESap026.xml:370]: the 20th

File: ES/ESap026.xml [hunk 3]
```diff
@@ -266,11 +365,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                   </origin>
+                  <provenance>Names of many individuals are mentioned in
+                     supplication formulas throughout the Ms. always in couples: e.g. 
+                     ʾabuna Gabra Māryām and Walatta Mikāʾel (s. <locus target="#5rc">5rc</locus>, 7rb, 13vb, 15va), 
+                     Gabra Krǝstos and Walatta
+                     Tǝnśāʾe (s. <locus target="#6ra">6ra</locus>, 8va), Takla Giyorgis and Walatta
+                     Māryām (s. <locus target="#6rc">6rc</locus>, 13va), Gabra Māryām and Walatta
+                     Krǝstos (s. <locus target="#6vc">6vc</locus>, 8ra), Gabra Ḫǝywät and ʿĀmata
```
Comment (DariaElagina) [inline ES/ESap026.xml:378]: Ḫǝywat

File: ES/ESdd039.xml [hunk 4]
```diff
@@ -58,15 +60,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <locus from="101va" to="110ra"/>
+                           <title type="complete" ref="LIT1932Mashaf#Nahase"/>
+                           
+                        </msItem>
+                        <msItem xml:id="ms_i1.13">
+                           <locus from="110ra" to="111rb"/>
+                           <title type="complete" ref="LIT1932Mashaf#Pagwemen"/>
+                           
+                        </msItem>
+                        <explicit type="supplication">
```
Comment (DariaElagina) [inline ES/ESdd039.xml:134]: xml:lang?

File: ES/ESdd039.xml [hunk 5]
```diff
@@ -58,15 +60,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        </msItem>
+                        <msItem xml:id="ms_i2.16">
+                           <locus from="133rb" to="134vb"/>
+                           <title type="complete" ref="LIT1932Mashaf#SNahase"/>                           
+                        </msItem>
+                        <msItem xml:id="ms_i2.17">
+                           <locus from="134vb" to="135rb"/>
+                           <title type="complete" ref="LIT1932Mashaf#SPagwemen"/>                           
+                        </msItem>
+                        <explicit type="supplication">
```
Comment (DariaElagina) [inline ES/ESdd039.xml:219]: xml:lang?

File: ES/ESdd039.xml [hunk 6]
```diff
@@ -207,92 +363,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <decoDesc>
 
                      <decoNote type="drawing" xml:id="d1">
-                        <locus target="#">ff. 137a-137aa: </locus>
+                        <locus target="#137ra"/>
                         <desc>Cross</desc>
                      </decoNote>
                      <decoNote type="frame" xml:id="d2">
-                        <locus target="#3ra">3ra</locus>, 3rb: two elaborate narrow coloured (black,
```
Comment (DariaElagina) [inline ES/ESdd039.xml:214]: maybe rather ornamentation? not a frame, see below as well

File: ES/ESgmg020.xml [hunk 7]
```diff
@@ -156,9 +151,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <item xml:id="e3">
+                           <desc>- Title of the work is written in the front board: Gǝṣṣāwe</desc>
+                        </item>
+                        <item xml:id="e4">
+                           <desc>
+                              - Crude note: <locus target="#55r">fol.  55r</locus>.</desc>
+                        </item>
+                        <item xml:id="e5">
+                           <desc> - Pen trials and doodles: fols. 55r, 55v, 56r, 56v.</desc>
                         </item>
```
Comment (DariaElagina) [inline ES/ESgmg020.xml:163]: locus?

File: ES/ESap026.xml [hunk 8]
```diff
@@ -47,38 +47,134 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                  <msItem xml:id="ms_i0">
+                     <locus from="5ra" to="200vc"/>
+                     <title type="complete" ref="LIT1932Mashaf#FirstHalf"></title>
+                     <note>The text is possibly copied from a printed edition, cp. the first
+                        chapter (kǝfǝl ʾand) of <bibl><ptr target="bm:MashafaGessawe1968"/></bibl></note>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESap026.xml:67]: it is valid

File: ES/ESap026.xml [hunk 9]
```diff
@@ -243,7 +339,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   </decoDesc>
                   <additions>
                      <list>
-                        <item xml:id="e1">- Written on folios of quire A, used as protective folios and sewn upside down, two excerpts from Mäṣḥafä Gǝṣṣawe “Lectionary for the Year” as follows: ጥር ፳፱-፴: fols. 2ra-2vc ኅዳር ፳፫-፳፬: fols. 3ra-3vc</item>
+                        <item xml:id="e1">-<desc type="GuestText"> Written on folios of quire A, 
+                           used as protective folios and sewn upside down, 
+                           two excerpts from <title ref="LIT1932Mashaf">Maṣḥafa gǝṣṣāwe</title> “Lectionary for the Year” 
+                           as follows: <q xml:lang="gez">ጥር ፳፱-፴</q>: fols. 2ra-2vc <q xml:lang="gez">ኅዳር ፳፫-፳፬</q>: fols. 3ra-3vc</desc></item>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESap026.xml:345]: I try to process E-S descriptions to make them readable as useable; marking with locus will mean having to restructure the whole passage as locus is visualized in the app at the top and not inside the line, so prefer for the moment to leave the descriptions of the extras as they are (in all E-S mss)

[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESap026.xml]: Done anyway

**Pull request merged**